### PR TITLE
Merge development branch into main branch

### PR DESCRIPTION
**Pull request by aristides1000**
The following changes were made:

- [x] [1pt] Create app and install basic dependencies - Setup (group task)
- [x] [1pt] Create website header with empty nav - Setup (group task)
- [x] [1pt] Create empty My profile - Setup (group task)
- [x] [1pt] Create basic structure for Rockets - Setup
- [x] [1pt] Create basic structure for Missions - Setup
- [x] [4pt] Fetch missions - Fetch data
- [x] [4pt] Fetch rockets - Fetch data
- [x] [4pt] Display missions - Lists render
- [x] [4pt] Display rockets - Lists render
- [x] [3pt] Implement rocket booking - Actions
- [x] [3pt] Implement mission joining - Actions
- [x] [1pt] Implement mission leaving - Actions
- [x] [1pt] Implement rocket booking cancelation - Actions
- [x] [1pt] Switch badges for Missions - Conditional components
- [x] [1pt] Switch badges for Rockets - Conditional components
- [x] [2pt] Display reserved rockets in My profile - Filters
- [x] [2pt] Display joined missions in My profile - Filters

### CODE REVIEW

File: src/pages/MyProfile.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+          </div>
+        </div>
+      </div>
+    </div>
+  );
+};
+
+// Profile.propTypes = {
+//   missions: Proptypes.shape({}).isRequired,
+// };
```
Comment (afanimartin) [inline src/pages/MyProfile.js:41]: Please remove this and any other comments in the codebase. Comments make our code look ugly.

**Review by afanimartin — CHANGES_REQUESTED** [review]
Hi @aristides1000  and @Laylo309, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

### Martin Afani
#### Slack: whiz25

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/pages/MyProfile.js (hunk 1 above)
Comment (aristides1000) [reply src/pages/MyProfile.js:41]: Thanks for your review Martin 👍

Comment (aristides1000) [reply src/pages/MyProfile.js]: I have already made the changes you have requested, can I submit a new review request?

**Review by DeliceLydia — APPROVED** [review]
# Congratulations🎉🎉

## Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**